frontend/src/features/entities/EntityList.tsx

```diff
@@ -14,6 +15,26 @@ interface Props {
   personalScores?: { [uid: string]: number };
 }
 
+const AvailableEntity = ({
+  children,
+  uid,
+}: {
+  children: React.ReactNode;
+  uid: string;
+}) => {
+  const [isAvailableOnYoutube, setIsAvailableOnYoutube] = useState(false);
+
+  useEffect(() => {
+    const img = new Image();
+    img.src = `https://i.ytimg.com/vi/${idFromUid(uid)}/mqdefault.jpg`;
+    img.onload = function () {
+      setIsAvailableOnYoutube(img.width !== 120);
+    };
```

Comment:
If an additional request is required to check the video availability, then it would be more reliable to use the HTTP status of the response with `fetch()`. The thumbnail of an unavailable video returns a 404 error.

Reply:
I've just tried to make it work with fetch but I ran in a cors policy wall.